### PR TITLE
feat(#33): add elastic APM

### PR DESCRIPTION
**Pull request by KRaffael**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Checklist

- [ ] I have read the [CONTRIBUTING.md](/CONTRIBUTING-template.md)
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no lint errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Only MIT licensed or MIT license compatible dependencies are used (e.g.: Apache2 or BSD)
- [ ] The code contains no credentials, personalized data or company secrets

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -71,7 +71,10 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/rs/zerolog v1.28.0
 	github.com/stretchr/testify v1.8.1
-	github.com/twmb/franz-go v1.9.0
+	github.com/twmb/franz-go v1.7.1
```
Comment (StephanHCB) [inline go.mod:74]: please don't downgrade franz-go (how did this happen?)

File: go.mod [hunk 2]
```diff
@@ -71,7 +71,10 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
```
Comment (StephanHCB) [inline go.mod:71]: not quite the correct line, but after merging please check in checkmarx if any extra ignores need to be added

File: internal/service/services/services_test.go [hunk 3]
```diff
@@ -131,6 +131,10 @@ func tstCreateValid() openapi.ServiceCreateDto {
 type MockConfig struct {
 }
 
+func (c *MockConfig) ElasticApmEnabled() bool {
+	return true
```
Comment (StephanHCB) [inline internal/service/services/services_test.go:135]: I believe this should be false so tests run without APM

**Review by StephanHCB — CHANGES_REQUESTED** [review]
revert franz-go downgrade and fix mock so that APM is DISabled.

File: go.mod (hunk 1 above)
Comment (KRaffael) [reply go.mod:74]: My vault, i have done a local downgrade because some dependencies cannot be downloaded there. I will upgrade to the right version.

**Review by StephanHCB — APPROVED** [review]

**Pull request merged**